### PR TITLE
Feat: add support for Duration for AWS Assume Roles in env0_aws_crede…

### LINKED ISSUE #710: Add support for Duration for AWS Assume Roles in env0_aws_credentials resource
**Is your feature request related to a problem? Please describe.**
env0 aws assume role credentials now support a `Duration` field.
please add support in TF provider.

see: https://docs.env0.com/reference/configuration-create-credential
duration-in-seconds

I believe default is 5 hours (18,000 secs)

**Describe the solution you'd like**

add a variable for duration (in seconds)

**Describe alternatives you've considered**
🤷 

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by TomerHeber**
…ntials resource

### Issue & Steps to Reproduce / Feature Request
resolves #710 

### Solution

1. Updated the schema.
2. Updated the API.
3. Updated the unit, acceptance and integration tests.

### CODE REVIEW

File: client/cloud_credentials_test.go [hunk 1]
```diff
@@ -80,7 +80,8 @@ var _ = Describe("CloudCredentials", func() {
 			mockOrganizationIdCall(organizationId)
 
 			payloadValue := AwsCredentialsValuePayload{
-				RoleArn: "role",
+				RoleArn:  "role",
+				Duration: 1,
```
Comment (sagilaufer1992) [inline client/cloud_credentials_test.go:84]: It's not one of the allowed values, shouldn't the test fail on this?

File: env0/resource_cost_credentials_test.go
Comment (sagilaufer1992) [inline env0/resource_cost_credentials_test.go:1]: I see only test that is updated with duration.
Can you test also creation with duration please?
And also create and update with illegal values?

If these cases are tested somewhere else please point me there

**Review by sagilaufer1992 — COMMENTED** [review]
Looks good. Have some comments about the tests.

File: client/cloud_credentials_test.go (hunk 1 above)
Comment (TomerHeber) [reply client/cloud_credentials_test.go:84]: This is enforced at the schema/resource level. Not the API level.

File: env0/resource_cost_credentials_test.go
Comment (TomerHeber) [reply env0/resource_cost_credentials_test.go:1]: Happy to add these.
I'll do it a bit later today.

Comment (TomerHeber) [reply env0/resource_cost_credentials_test.go:1]: @sagilaufer1992 sorry for the delay.
I have added the requested use-cases.

Comment (sagilaufer1992) [reply env0/resource_cost_credentials_test.go:1]: LGTM

**Review by sagilaufer1992 — APPROVED** [review]
LGTM, cases added as requested in other comment

**Pull request merged**